src/AWS.Deploy.Orchestration/CDK/CDKBootstrapTemplate.yaml

```diff
@@ -489,7 +490,7 @@ Resources:
       Type: String
       Name:
         Fn::Sub: /cdk-bootstrap/${Qualifier}/version
-      Value: "12"
+      Value: "13"
```

Comment:
You need to also update the [Constant]( https://github.com/aws/aws-dotnet-deploy/blob/5b78ecaec217908c19d92fabbdfcf1963821194a/src/AWS.Deploy.Constants/CDK.cs#L19)

If you are filling ambitious you get rid of the constant and parse the version from the bootstrap yaml file so we don't have to remember to update both places in the future.

Reply:
Done